project/BUILD

```diff
@@ -0,0 +1,11 @@
+load("@rules_java//java:defs.bzl", "java_plugin")
```

Comment:
Please add the copyright header here too.

Reply:
Done ✅